bertopic/backend/_utils.py

```diff
@@ -126,8 +134,14 @@ def select_backend(embedding_model,
                                 "Else, please select a language from the following list:\n"
                                 f"{languages}")
 
-        # Only for light-weight installation
-        except ModuleNotFoundError:
+        # A ModuleNotFoundError might be a lightweight installation
+        except ModuleNotFoundError as e:
+            if e.name != "sentence_transformers":
+                # Error occurred in a downstream module, probably not a lightweight install
+                raise e
+            # Whole sentence_transformers module is missing, probably a lightweight install
+            if verbose:
+                logger.info("Automatically selecting lightweight scikit-learn embedding backend as sentence-transformers appears to not be installed.")
```

Comment:
The if statement is not needed since you already set the logging level right?

Reply:
I think I only did this because that's how it's done in a few other places e.g. `BERTopic._extract_topics()`. If that's unnecessary I can remove

Reply:
Ah, you're right! Let's keep it for now and when the logger updates I might change it.